targets/arduino_101/src/libc_support.c

```diff
@@ -0,0 +1,105 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
Why is this file needed? It is a copy-paste of jerry-libc functions, but ports can call jerry-libc functions. Did I miss something here? :)